packages/react/src/components/Checkbox/Checkbox-test.js

```diff
@@ -85,7 +83,7 @@ describe('Checkbox', () => {
     const label = wrapper.find('label');
     expect(label.length).toEqual(1);
     const span = wrapper.find('span');
-    expect(span.hasClass(`${prefix}--visually-hidden`)).toEqual(true);
+    expect(span.hasClass(`wfp--visually-hidden`)).toEqual(true);
```

Comment:
Can you please check why not use the `prefix` here?

Reply:
Previously, it was importing prefix from a wrong file which was causing it to throw an error. I wanted to point to the right file but later changed my mind because all the other test files are referencing `wfp`. I will have time to go back into the test to use the prefix later. But for the mean time I just went with the usual format.